wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -793,16 +793,15 @@ def i18n_enabled():
 
 
 @register.simple_tag
-def locales():
-    return json.dumps(
-        [
-            {
-                "code": locale.language_code,
-                "display_name": force_str(locale.get_display_name()),
-            }
-            for locale in Locale.objects.all()
-        ]
-    )
+def locales(serialize=True):
```

Comment:
If we're sure that this isn't used anywhere else, we can remove the parameter and turn this into a normal function (not a template tag) that's called in the `wagtail_config` tag, and remove the `json.dumps()` call.

Similar to `get_js_translation_strings()`.

Reply:
yeah, I think it's only used by the page explorer, I actually think it should not be in the global 'config' at all, instead maybe passed into the sidebar data (that in turn gets passed to the page explorer) or similar.

https://github.com/wagtail/wagtail/blob/main/wagtail/admin/ui/sidebar.py#L130